src/dataaggregator/aggregator/aggregator.go

```diff
@@ -19,63 +19,67 @@ type Aggregator struct {
 	appChan            chan *appmetric.AppMonitor
 	policyPoller       *PolicyPoller
 	metricPollerCount  int64
-	metricPollerArray  []*MetricPoller
+	MetricPollerArray  []*MetricPoller
```

Comment:
No. This should remain private. We don't expose things just so it makes testing easier. There are other ways.